pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -114,6 +114,14 @@ func ShootWantsAlertmanager(shoot *gardenv1beta1.Shoot, secrets map[string]*core
 	return false
 }
 
+// ShootIgnoreAlerts checks if the alerts for the annotated shoot cluster should be ignored.
+func ShootIgnoreAlerts(shoot *gardenv1beta1.Shoot) bool {
+	if ignore, ok := shoot.Annotations[common.GardenIgnoreAlerts]; ok && (ignore == "true") {
+		return true
+	}
+	return false
```

Comment:
Please do it similar to https://github.com/gardener/gardener/blob/e701336bbd78cc37817e32de3ebbdcce790920ae/pkg/controllermanager/controller/shoot/shoot_utils.go#L91-L98 to also allow values others than "true"